src/DynamoCore/Extensions/LinterExtensionBase.cs

```diff
@@ -151,22 +155,30 @@ private void InitializeRules()
 
         #region Extension Lifecycle
 
+        ///<inheritdoc/>
         public virtual void Ready(ReadyParams sp)
```

Comment:
I would say to seal all the virtuals unless absolutely necessary and we can decide to expose them later

Reply:
i think there is some issues with not making these `virtual`, we would essentially remove all entry points for the implementer. For example, I'm using `Ready` to add the linter rules in the `TestLinterExtension` not sure where or how we would add these if these are not accessible for the implementer.

Unless we add a new `abstract List<LinterRule> AddLinterRules()` to the base that gets called in `Ready`?

Reply:
Yeah, I think we can keep this one